src/credentials/tests/TestDeviceAttestationCredentials.cpp

```diff
@@ -212,6 +212,16 @@ TEST_F(TestDeviceAttestationCredentials, TestDACVerifierExample_AttestationInfoV
     ASSERT_NE(example_dac_verifier, nullptr);
     EXPECT_NE(default_verifier, example_dac_verifier);
 
+    example_dac_verifier->EnableCdTestKeySupport(false);
+    EXPECT_FALSE(example_dac_verifier->AreVerboseLogsEnabled());
+    EXPECT_FALSE(example_dac_verifier->IsCdTestKeySupported());
```

Comment:
What sets that to false?  The verifier defaults it to true, right?

Reply:
I set it to false above, and users of example DAC verifier have to set it to false manually. I do feel since the start it's the wrong API, but I got asked initially to "not break anyone" since everyone who  complained at the time wanted to trust CD test keys most of the time (for dev).